data/news/mediaAssetPage/business-23249447.json

```diff
@@ -0,0 +1 @@
xml","type":"paragraph"},{"text":"-- WE DON'T NEED TO SUPPORT THIS BUT SHOULD RENDER IT AS SOMETHING ELSE-- ","markupType":"plain_text","type":"heading"},{"text":"This is an introduction","role":"introduction","markupType":"plain_text","type":"paragraph"},{"text":"This is a heading ","markupType":"plain_text","type":"heading"},{"text":"this is transmission info ","role":"transmission-info","markupType":"plain_text","type":"paragraph"},{"text":"This is a footer ","role":"footer","markupType":"plain_text","type":"paragraph"},{"numbered":true,"items":[{"text":"I'm line 1 in a numbered list","markupType":"plain_text","type":"listItem"},{"text":"<bold>I'm line 2 in a numbered list I'm in bold </bold>","markupType":"candy_xml","type":"listItem"},{"text":"<italic>I'm line 3 in a bullet list I'm in italics </italic>","markupType":"candy_xml","type":"listItem"},{"text":"<link><caption>I'm line 4 in the bullet list and I'm a link</caption><altText>This is the alt text for the link in the bullet list </altText><url href=\"https://bbc.com/pidgin\" platform=\"highweb\"/></link>","markupType":"candy_xml","type":"listItem"}],"type":"list"},{"text":"This is text where someone has added in &lt;bold&gt;markup&lt;/bold&gt;","markupType":"plain_text","type":"paragraph"}]},"promo":{"headlines":{"shortHeadline":"New Media Team Begins to understand CPS","headline":"New Media Team Begins to understand CPS Full Headline"},"locators":{"assetUri":"/news/av/business-23249447","cpsUrn":"urn:bbc:content:assetUri:news/av/business-23249447","curie":"http://www.bbc.co.uk/asset/6faf0ed9-bbd5-754a-9719-cec3a46f50c6"},"summary":"New Media Team Begins to understand CPS Summary","timestamp":1569235774497,"passport":{},"media":{"id":"p01ggx4d","subType":"clip","format":"video","title":"Demolition","synopses":{"short":"A bus driver ruined a perfect shot"},"imageUrl":"ichef.test.bbci.co.uk/images/ic/$recipe/p01ggx88.jpg","embedding":true,"advertising":true,"caption":"A bus driver ruined a perfect shot","versions":[{"versionId":"p01ggx4g","types":["Original"],"duration":38,"durationISO8601":"PT38S","warnings":{},"availableTerritories":{"uk":true,"nonUk":true},"availableFrom":1518788866000}],"type":"media"},"indexImage":{"id":"63722155","subType":"index","href":"http://b.files.bbci.co.uk/D752/test/_63722155_p01ggx88.jpg","path":"/cpsdevpb/D752/test/_63722155_p01ggx88.jpg","height":360,"width":640,"altText":"Keyframe #1","copyrightHolder":"BBC","type":"image"},"id":"urn:bbc:ares::asset:news/av/business-23249447","type":"cps"},"relatedContent":{"section":{"subType":"index","name":"Business","uri":"/news/business","type":"simple"},"site":{"subType":"site","name":"BBC News","uri":"/news","type":"simple"},"groups":[{"type":"more-media-asset-pages","promos":[{"locators":{"assetUri":"/news/10301118","cpsUrn":"urn:bbc:content:assetUri:/news/10301118"},"title":"AV content promotion hyperpuff - importers - LIVE","timestamp":1279018179000,"id":"urn:bbc:ares::asset:news/10301118","type":"hyper"}]}]}}
```

Comment:
Could you run this through a linter please

Reply:
Oops, this usually happens automatically - linted this (and other) files